packages/configuration-backend/src/service.ts

```diff
@@ -95,28 +97,59 @@ export const getRestrictedServiceConfiguration = async (p: {
   }
 }
 
-export const getMerchantDetailsForOrder = async (p: { namespace: string }) => {
-  const { namespace } = p
-  const allConfiguration = await getAllPublicServiceConfiguration({ namespace })
-
-  return allConfiguration.reduce((acc, cur) => {
-    const { configurationKey: k, configurationValue } = cur
-    if (
-      k === 'merchantName' ||
-      k === 'merchantStreet' ||
-      k === 'merchantZip' ||
-      k === 'merchantCity' ||
-      k === 'merchantEmail' ||
-      k === 'merchantPhone' ||
-      k === 'merchantShopId' ||
-      k === 'merchantUrl' ||
-      k === 'merchantTermsOfServiceUrl' ||
-      k === 'merchantBusinessId'
-    ) {
-      acc[k] = configurationValue
+export const getMerchantDetailsForOrder = async (order: {
+  namespace: string
+  items: OrderItem[]
+}) => {
+  const { namespace } = order
+  let merchantId = parseMerchantIdFromFirstOrderItem(order) || ''
+  return getMerchantDetailsWithNamespaceAndMerchantId(namespace, merchantId)
+}
+
+export async function getReducedServiceConfigurationsForNamespace(
```

Comment:
We should probably stick with ES6 arrow function syntax for function definitions, because it is already used everywhere else in the code. Could you change this to be as a arrow function just so that the syntax is uniform across the whole ExpAPI?